README.md

```diff
@@ -101,6 +101,200 @@ any zone.
 
 You can define an arbitrary number of drag items.
 
+Analytics Events
```

Comment:
@Kelketek As a general comment/question, would it make sense to exclude

> "common" fields that are not interesting in this context

from the examples below? (I.e., use the *first* example from the [event design page](https://openedx.atlassian.net/wiki/display/AN/DnDv2+XBlock+Event+Design) for each event)